### PR TITLE
DEVELOPER-3440 - Added configuration for querying the DCP tags

### PR DESCRIPTION
**Pull request by Dantheman720**

### CODE REVIEW

**Comment by Dantheman720** [comment]
To test this you will need to create a new topic page in drupal. I used the tags Atomic, Containers and CDK as the data tags in the input box (at the bottom). Once you publish your changes, you should see white cards at the bottom populating from the rest query.

File: _docker/drupal/drupal-filesystem/web/themes/custom/rhd/templates/topic/field--node--field-data-tags--topic.html.twig [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+{% for item in values %}
+    {% if not loop.last %}
+        {% set data = data ~"'"~ item|trim ~"'" ~ ',' %}
+    {% else %}
+        {% set data = data ~ "'" ~ item|trim ~ "'" ~ "]" %}
+
+    {% endif %}
+{% endfor %}
+
+<div id="topic-resources" class="row" data-tags={{ data }}>
```
Comment (LightGuard) [inline _docker/drupal/drupal-filesystem/web/themes/custom/rhd/templates/topic/field--node--field-data-tags--topic.html.twig]: It seems like some twig filters would work best here:

http://twig.sensiolabs.org/doc/filters/index.html

I'm thinking `<div id="topic-resources" class="row" data-tags={{content|split(',')|json_encode}}>` would really be all you'd need to do here. I'll try it locally and if it works I'll make the change here.

**Comment by LightGuard** [comment]
retest this please

**Comment by LightGuard** [comment]
@Dantheman720 what do you think about those comments?

**Comment by Dantheman720** [comment]
retest this please

**Comment by RubyGemTSL** [comment]
retest this please

**Comment by Dantheman720** [comment]
retest this please

**Comment by RubyGemTSL** [comment]
I would hold on retesting until #1522 is merged.

**Comment by Dantheman720** [comment]
Sounds good. I'll keep an eye on https://github.com/redhat-developer/developers.redhat.com/pull/1522

**Comment by robpblake** [comment]
retest this please

**Comment by Dantheman720** [comment]
@LightGuard - I don't know that the config imports are working properly on the PRs. I don't see the data tags field (which still works locally here). I could ssh into the box and import? Perhaps that would work?

**Comment by LightGuard** [comment]
@Dantheman720 Yeah, worth doing I think.

**Comment by Dantheman720** [comment]
@LightGuard sounds good

**Pull request merged**